src/Controls/tests/TestCases.HostApp/Issues/Issue17390/MainTabPage.xaml.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Maui.Controls.Sample.Issues
+{
+	public partial class MainTabPage : ContentPage
+	{
+		public MainTabPage()
```

Comment:
For classes that are specific to a given issue can we make this internal classes of the issue or put them in a namespace matching the issue number? 

Otherwise we're going to really clutter up the main Issues namespace